packages/frontEnd/src/views/blogList/articles/ArticleCard.tsx

```diff
@@ -0,0 +1,73 @@
+import { IoPerson } from "react-icons/io5";
+import { MdAccessTimeFilled } from "react-icons/md";
+import { FaLongArrowAltRight } from "react-icons/fa";
+
+interface ArticleCardProps {
+  img: string;
+  title: string;
+  date: string;
+  avatar: string;
+  category: string;
+  author: string;
+  content: string;
+}
+
+export const ArticleCard = ({
+  img,
+  title,
+  date,
+  avatar,
+  category,
+  author,
+  content,
+}: ArticleCardProps) => {
```

Comment:
I think you can use a switch here instead of regular is else.
```
switch (category) {
  case 'Marketing':
    return "bg-green-600";
    break;
  case 'Advertise':
    return "bg-purple-blog"; 
    break;
  default:
    return "bg-yellow-600";
}

```